envoy/api/v2/route/route.proto

```diff
@@ -768,6 +768,17 @@ message RateLimit {
   repeated Action actions = 3 [(validate.rules).repeated .min_items = 1];
 }
 
+// [#not-implemented-hide:]
+// Specifies the int64 lower and upper bounds of a range.
+// Header route matching will be performed if the header's value lies within this range (inclusive).
+message HeaderValueRange {
+  // Low key, inclusive
+  google.protobuf.Int64Value low_key = 1;
+
+  // High key, inclusive
+  google.protobuf.Int64Value high_key = 2;
```

Comment:
Nit: prefer just making these `high/low` or `high_value/low_value`. Key here is confusing.

Reply:
yes, I ll change this to low/high.